omero/sysadmins/server-upgrade.rst

```diff
@@ -324,6 +324,13 @@ up to date to ensure that security updates are applied:
       $ pip install --upgrade 'omero-py>=\ |version_py|'
 
 
+Server certificate
+^^^^^^^^^^^^^^^^^^
+Since OMERO 5.6.2,  the recommended way to ensure that all OMERO server installations have, at minimum, a self-signed certificate is to use the `OMERO server certificate management plugin <https://github.com/ome/omero-certificates>`_. The plugin will generate or update your self-signed certificates and configure the OMERO.server. Each time, you run the command to generate or update the certificates is run, the server needs to be restarted.
```

Comment:
` Each time, you run the command to generate or update the certificates is run, the server needs to be restarted.`

I don't understand this sentence

Reply:
I changed the sentence half way through and did not read it again.
Maybe something like ``for the configuration to take effect, the server needs to be restarted`` or we omit it